src/main.rs

```diff
@@ -436,7 +436,9 @@ fn print_tree<'a>(
     let mut visited_deps = HashSet::new();
     let mut levels_continue = vec![];
 
-    let node = &graph.graph[graph.nodes[&package]];
+    let package = graph.nodes.get(package)
+      .expect(&format!("package {} not found, try specifying an explicit --target", package));
```

Comment:
Let's just have this function return a `CargoResult` instead and make a proper error here.

Reply:
👍 good stuff, didn't realize how easy that was going to be